driver/mounter_test.go

```diff
@@ -0,0 +1,110 @@
+package driver
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+type testAttachmentValidator struct {
+	readFileFunc     func(name string) ([]byte, error)
+	evalSymlinksFunc func(path string) (string, error)
+}
+
+func (av *testAttachmentValidator) readFile(name string) ([]byte, error) {
+	return av.readFileFunc(name)
+}
+
+func (av *testAttachmentValidator) evalSymlinks(path string) (string, error) {
+	return av.evalSymlinksFunc(path)
+}
+
+func Test_mounter_IsAttached(t *testing.T) {
+	t.Run("could not evaluate the symbolic link", func(t *testing.T) {
+		m := &mounter{
+			log:      logrus.NewEntry(logrus.New()),
+			kMounter: nil,
+			attachmentValidator: &testAttachmentValidator{
+				readFileFunc: nil,
+				evalSymlinksFunc: func(path string) (string, error) {
+					return "", errors.New("error")
+				},
+			},
+		}
+
+		_, err := m.IsAttached("")
+		if err == nil {
```

Comment:
We should check for a specific error string (`"error"` in this case) so avoid false positives as I mentioned in a previous comment.

Also applies for other test cases.